### ISSUE TITLE
Substring filter?

### ISSUE THREAD

**Issue by gstf** [mapbox/mapbox-gl-js#2499]
**mapbox-gl-js version**: 17.0.0

Is there a way to filter features by substring matching?

Example, layer (id: test) with this source.
```
{
  "properties": { "name": "Hello" }
  "geometry": ...
}
```

The following filter would show this feature.
```
map.setFilter('test', ['<what to put here?>', 'name', 'el'])
```

**Comment by jfirebaugh** [comment]
Not currently. This has previously been requested in https://github.com/mapbox/mapbox-gl-style-spec/issues/438.

**Comment by Topduck** [comment]
Has this issue been reopened, or looked into?  I could really use this in my projects.

**Comment by mourner** [comment]
@Topduck it was implemented in v1.6.0 as the `in` expression.
